CMSgov/design-system
# [NO-TICKET] Sketch token importer updates

## Summary

- Updates the sketch token importer to organize tokens into categories based on their name
- Removes submenu under plugins for easier access!
- Fixes regex to be more accepting of various token declarations
- updated `.eslintignore` to ignore the compiled sketch plugin

## How to test

Can build / import tokens with it if you like.

- run `yarn build themes json` from the design-system-tokens package
- open sketch, new document
- double click on the `cmsds-token-importer.sketchplugin` file to install the plugin
- under plugins, choose `CMSDS Color Token Importer`
- choose one of the core JSON files in the `design-system-tokens/dist` folder (not a `-component` file)
- check out your swatches!

![image](https://user-images.githubusercontent.com/783820/171244535-22dee4bc-d225-4f35-9d84-d925b4e048b2.png)
